Source/WebCore/Modules/mediasource/MediaSource.h

```diff
@@ -251,8 +251,8 @@ class MediaSource
     uint64_t m_logIdentifier { 0 };
 #endif
     std::atomic<uint64_t> m_associatedRegistryCount { 0 };
-    Ref<MediaSourceClientImpl> m_client;
     RefPtr<MediaSourcePrivate> m_private;
+    Ref<MediaSourceClientImpl> m_client;
```

Comment:
Could we add a comment stating that this data member MUST be after m_private? Otherwise, it is very easy to regress.